templates/docs/patterns/pricing-block/index.md

```diff
@@ -62,17 +63,25 @@ A single pricing block will take all the available space.
 View example of the single pricing block
 </a></div>
 
+## Block with a normal rule
+
+A pricing block with the <a href="https://vanillaframework.io/docs/patterns/rule#default">default variant</a> of the rule
```

Comment:
```suggestion
A pricing block with a<a href="https://vanillaframework.io/docs/patterns/rule#default">default rule</a> at the top
```

I would suggest we use "default rule" instead of "default variant of the rule" as it makes it clearer that we are referring to the rule component itself, not a variant of this pattern, and it's also more consistent with the language you've used for the other rule variants